README.md

```diff
@@ -23,7 +23,7 @@ Welcome to the **Versify**! This repository contains code to parse and analyze t
 1. Clone this repository:
     ```sh
     git clone https://github.com/Namit2111/bible-verse-finder.git
-    cd bible-verse-finder
+    cd bible-verse-finder/backend
```

Comment:
why is this updates to /backend ? the title says clone this repo

Reply:
With the new file structure that was introduced in previous PRs @Namit2111, all backend codes are now in a folder called `backend`, this title says "Clone this repository" but looking at the next lines after this, it has info about starting the python app. If we're not in the folder there will be issues.